app/javascript/packages/address-search/index.tsx

```diff
@@ -1,3 +1,4 @@
+import { Alert } from '@18f/identity-components';
```

Comment:
What's the rubric for deciding whether to publish a package on its own (as we did with i18n) vs. re-exporting (as we're doing here)?

Reply:
>I'll publish it under a new package.

If you mean "new" as in, the newly-published (but existing in this project) `@18f/identity-components`, then 👍 from me.